index.html

```diff
@@ -23,93 +23,105 @@
-      GNU Emacs is <strong>Free Software</strong>, free as in free beer and
-      free as in free speech. The original Emacs was written by <em>St.
-      iGNUcius himself</em>&nbsp;&mdash; <strong>Richard Stallman</strong>.
+      <img src="img/emacs-logo.png" alt="Emacs" class="left" /> GNU
+      Emacs is <strong>Free Software</strong>, both free as in free
+      beer and free as in free speech. The original Emacs was written
+      by <em>St. iGNUcius himself</em>&nbsp;&mdash; <strong>Richard
+      Stallman</strong>.
     </p>
     <p>
       Not only does it give the user
       the <a href="https://www.gnu.org/philosophy/free-sw.html">four
-        essential freedoms</a>, its architecture is built in such a way
+      essential freedoms</a>, its architecture is built in such a way
       that you have the power to mold it, change it and make it comply
       to your computering needs.
     </p>
   </div>
 </section>
 <section id="_why" class="gray">
   <div>
-    <h2>Why using Emacs?</h2>
-    It's a very powerful text processor, allowing you to manipulate
-    blocks of text, paragraphs, words, finding, replacing, using
-    regular expressions, keyboard macros and more.
+    <h2>Why use Emacs?</h2>  Emacs is a very powerful text processor,
+    giving you the power to manipulate documents quickly and
+    efficiently. You can easily move through and edit paragraphs,
+    sentences, words, and logical blocks; blaze through text using
+    powerful search tools; and easily edit thousands of lines at once
+    using regular expressions, keyboard macros and more.
     <h3>Colorful text editor</h3>
     <p>
-      Emacs can be adapted in every possible way, including its looks.
-      You can <a href="http://bzg.fr/emacs-strip-tease.html">strip it</a>,
-      choose between several themes with <code>M-x
-      load-theme</code>, download one from another happy Emacs
-      user or create your own.
+      Emacs can be customized in every conceivable way, including its
+      looks.  You
+      can <a href="http://bzg.fr/emacs-strip-tease.html">strip it
+      down</a>, choose between dozens of easy to install themes
+      with <code>M-x load-theme</code>, or even create your own and
+      share it with your friends.
     </p>
     <h3>Et tu, Programmer?</h3>
     <p>
-      There are tools for probably every programming language out
-      there. Lisp, Ruby, Python, PHP, Java, Erlang, JavaScript, C,
-      C++, Prolog, Tcl, AWK, PostScript, Clojure, Scala,
-      Perl, Haskell, Elixir all of these languages and more are
-      supported in Emacs.
+      There are tools for every programming language out there. Lisp,
+      Ruby, Python, PHP, Java, Erlang, JavaScript, C, C++, Prolog,
+      Tcl, AWK, PostScript, Clojure, Scala, Perl, Haskell, Elixir all
+      of these languages and more are supported in Emacs. Because of
+      the powerful Lisp core, Emacs is easy to extend to new languages
```

Comment:
easy to extend **for** new languages?  I'm not sure, but it seems slightly better to me.  With `to` it misses a verb... easy to extend to **add support** or something.  What do you think?